packages/next-app/src/app/internalmanager/(components)/TokenSelect.tsx

```diff
@@ -1,21 +1,21 @@
 "use client";
 
+import { SingleInternalBalanceQuery } from "@balancer-pool-metadata/gql/src/balancer-internal-manager/__generated__/Mainnet";
 import { ChevronDownIcon } from "@radix-ui/react-icons";
 import Image from "next/image";
+import Link from "next/link";
+import { tokenLogoUri } from "public/tokens/logoUri";
 import React, { useEffect, useState } from "react";
 import { useAccount, useNetwork } from "wagmi";
 
 import genericTokenLogo from "#/assets/generic-token-logo.png";
 import { Dialog } from "#/components/Dialog";
 import { Input } from "#/components/Input";
 import Table from "#/components/Table";
-import {
-  SelectedToken,
-  useInternalBalance,
-} from "#/contexts/InternalManagerContext";
 import { impersonateWhetherDAO, internalBalances } from "#/lib/gql";
 import { moralisGetTokens } from "#/lib/moralis";
 import { refetchRequest } from "#/utils/fetcher";
+import { ArrElement, GetDeepProp } from "#/utils/getTypes";
 
 interface tokenItem {
   token_address: string;
```

Comment:
Because of Moralis, but this will be taken out soon. 
<img width="631" alt="image" src="https://github.com/bleu-studio/balancer-pool-metadata/assets/63298341/4e959cab-556b-40f6-a0cb-0408bdb02258">